common/app/common/models/AttributeBean.java

```diff
@@ -0,0 +1,62 @@
+package common.models;
+
+import common.contexts.UserContext;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.products.attributes.Attribute;
+import io.sphere.sdk.products.attributes.AttributeAccess;
+import io.sphere.sdk.products.attributes.AttributeExtraction;
+import io.sphere.sdk.producttypes.MetaProductType;
+
+public class AttributeBean extends Base {
+    private String name;
+    private String key;
+    private String value;
+
+    public AttributeBean() {
+    }
+
+    public AttributeBean(final Attribute attribute, final MetaProductType metaProductType, final UserContext userContext) {
```

Comment:
We later need to adjust this since I want to deprecate MetaProductType. But for this pr ok.